### ISSUE TITLE
Vets-api terminal access for Noah Stern

### ISSUE THREAD

**Issue by nfstern02** [department-of-veterans-affairs/va.gov-team#59190]
### E-QIP Transmittal Date

- [x] Check this box if you have an E-QIP transmittal date.

### Your Name

Noah Stern

### Your GitHub handle (terminal access is managed through GitHub)

@nfstern02

### Team, Role, and Company of the target individual

VFEP Sustainment, Ruby/Rails developer, GCIO

### Product Manager (PM) name and email

Kara Ciprich, Kara.Ciprich@va.gov

### Product Owner (PO) name and email

Darla van Nieukerk, Darla.vanNieukerk@va.gov

### Your dsvagovcloud AWS user name (if applicable)

Noah.Stern

### ArgoCD terminal access requested for the following environments

- [ ] dev
- [ ] staging
- [ ] sandbox
- [X] production (if requesting, justify prod access in 'Additional Notes' section)

### COR Name (leave "None" if you listed an AWS user name above)

None

### Vendor Onboarding Representative (VOR) Name (leave "None" if you listed an AWS user name above)

None

### Access Expiration

ongoing

### Additional Notes

Requesting **read-only** terminal access for ARGOCD Vets-Api production. I would like to have this access so I can confirm production code pushes worked successfully by checking for lines of code that got pushed.

We have had a couple of issues with production releases not going off correctly and need to get out in front of them right away instead of getting notified through users having issues.

Thx,
Noah

### User must exist in a roster before vets-api terminal access can be granted

- [x] Search for user on the [VFS Team Roster](https://docs.google.com/spreadsheets/d/11dpCJjhs007uC6CWJI6djy3OAvjB8rHB65m0Yj8HXIw/edit?folder=0ALlyxurHpUilUk9PVA#gid=2042046665)
- [ ] Or search for user on the [Platform Team Roster](https://vfs.atlassian.net/wiki/spaces/AP/pages/1908834623/Platform+Roster)
- [ ] If user is on a VFS team but not in the VFS Team Roster, add the 'NOT YET' label and instruct them to start the [Platform orientation process](https://depo-platform-documentation.scrollhelp.site/getting-started/Platform-Orientation.1877344532.html)
- [ ] If user is on a Platform team but not on the Platform Team Roster in Confluence, add the 'NOT YET' label and instruct them to reach out to their Product Manager to be added
- [x] If they included an AWS user name, check to see if their user name is listed in [iam_users.tf](https://github.com/department-of-veterans-affairs/devops/blob/master/terraform/environments/global/iam_users.tf)
- [ ] Comment in this issue saying which roster the user is listed in and confirming their AWS user name (if applicable)

**Comment by nfstern02** [comment]
Since read-only access is not an option, would like to request terminal access.

**Comment by jknipes** [comment]
@nfstern02 AWS user name has been confirmed. Access to `prod` is pending approval by OCTODE, and we'll update you as soon as it's available.

**Comment by BillChapmanUSDS** [comment]
I can approve the standard 3 month time limited access here but I need the E-QIP date added here as per our latest process.

I want to give warning ahead of time that the reason for production access likely does not meet best practices and might not be allowed after this 3 month expires. Please ensure your pipelines are adequately logging your deployments to the appropriate locations for observability and tracking.

**Comment by ryan-mcneil** [comment]
@BillChapmanUSDS We received an email from a Project Coordinator, on behalf of Noah:

```
Good Afternoon Joe,
I'm reaching out to you on behalf of Noah Stern. He mentioned that he's trying to get access to some production servers but being asked for a transmittal notice screenshot. Unfortunately his background investigation was completed prior to his joining his current contract so we never received a transmittal notice, we only got a Certificate of Reciprocity. Is there anything besides a Transmittal notice that will help you expedite Noah's access to the server he's requesting access to? I can tell you that his T4 BI was Closed as Favorable on 12/03/2020. If there's a form or a certificate or something that I can supply to help with this, please let me know.
Thanks,
Nik Aberle (Contractor)
GovernmentCIO, Project Coordinator 2
On-Off Boarding Liaison (LIPAS, VLM-NG, VR&E PSES)
Email:[nikolas.aberle@va.gov](mailto:nikolas.aberle@va.gov)
Email: [nikolas.aberle@govcio.com](mailto:nikolas.aberle@govcio.com)
```

This seems to fall into one of those gaps we've recently discussed. As this shows that that Noah has already received a favorable adjudication, I assume that is enough for you (or is something you can confirm). I'll let you give your final approval before granting the 3 months. Thanks!

**Comment by BillChapmanUSDS** [comment]
Hey Ryan,  The transmittal email is really only necessary for folks that don't have an adjudicated clearance, in this case I think we can accept any proof of adjudicated clearance or even just a federal PO saying "I am validating they are clear" I will accept this note from the product coordinator in this case. 

Production approved for 3 month access.

**Comment by jknipes** [comment]
@nfstern02 you have been approved for `prod` access until Sept. 20. 
Please check out the following documentation https://depo-platform-documentation.scrollhelp.site/developer-docs/vets-api-on-eks#VetsAPIonEKS-TerminalAccess

**Comment by jknipes** [comment]
Remind [#platform-tech-team-support](https://dsva.slack.com/archives/C02RLAUM2LV) on “remove access to PROD vets-api terminal for user [@nfstern02](https://github.com/department-of-veterans-affairs/va.gov-team/issues/59190)” at 9AM Wednesday, September 20th.

**Comment by jknipes** [comment]
@nfstern02 Per an update to our access policy, your ArgoCD/vets-api Production access has been extended for 3 months, expiring 12/20/23. Please set a reminder for yourself to return to this Issue and re-request access within 1 week of the expiration date. You'll receive 1 reminder with a 72 hour window to respond. Failure to do so will result in lost access and will require you to fill out a new request.

**Comment by nfstern02** [comment]
@jknipes Can you extend for another 3 months? It's expiring on the 20th of December. Thx, Noah.

**Comment by bdech** [comment]
@LindseySaari @ericboehs can one of you please give us a :thumbsup: to extend their production access?

**Comment by bdech** [comment]
@nfstern02 Your `Production` access has been extended until 06/20/24. Please set a reminder for yourself to return to this Issue and re-request access within 1 week of the expiration date. You'll receive 1 reminder with a 72 hour window to respond. Failure to do so will result in lost access and will require you to fill out a new request.

**Comment by bdech** [comment]
support slack reminder [here](https://dsva.slack.com/archives/C02RLAUM2LV/p1702921783081079)

**Comment by nfstern02** [comment]
Thank you very much Brandon!!!

**Comment by lindsay-insco** [comment]
@nfstern02 @simiadebowale Your vets-api terminal Production access has expired.

Would you like to request an extension? You have 72 hours to respond, or your
access will be revoked and you will need to fill out a new request

**Comment by nfstern02** [comment]
Yes, I would like to request an extension. Did Gregg Puhala's access expire too? He would need his extended as well. Thanks for letting me know Lindsay.

**Comment by lindsay-insco** [comment]
@nfstern02 let me double-check Gregg's!

@LindseySaari @ericboehs can one of you please give us a 👍 to extend their production access?

**Comment by lindsay-insco** [comment]
Slack reminder [here](https://dsva.slack.com/archives/C06CY71PCAX/p1726151325432099)

**Comment by bdech** [comment]
@nfstern02  your Production access has been extended for 6 months,expiring 09/12/25.

Please set a reminder for yourself to return to this Issue, reopen it, and request an extension within 1 week of the expiration date. You'll receive 1 reminder with a 48 hour window to respond. Failure to respond will result in your access being revoked and will require you to fill out a new request.

**Comment by bdech** [comment]
support slack reminder [here](https://dsva.slack.com/archives/C06CY71PCAX/p1741704582782959)

**Comment by nfstern02** [comment]
Thank you Brandon
